masters.tf

```diff
@@ -184,11 +184,12 @@ resource "aws_security_group_rule" "ingress-worker-to-master" {
 }
 
 resource "aws_security_group_rule" "master-ssh" {
+  count                    = "${length(var.ssh_security_group_ids)}"
   type                     = "ingress"
   from_port                = 22
   to_port                  = 22
   protocol                 = "tcp"
-  source_security_group_id = "${var.ssh_security_group_id}"
+  source_security_group_id = "${element(var.ssh_security_group_ids), count.index}"
```

Comment:
Shouldn't this be `${element(var.ssh_security_group_ids, count.index)}`?

Reply:
yes, tyvm